prediction_market_agent/agents/replicate_to_omen_agent/omen_replicate.py

```diff
@@ -226,7 +227,7 @@ def omen_unfund_replicated_known_markets_tx(
         # Optionally, if `saturation_above_threshold` is provided, skip markets that are not saturated to leave some free money motivation for agents.
         if (
             saturation_above_threshold is not None
-            and not market.is_resolved
+            and market.is_open
```

Comment:
Edge-case, if market is closed, but isn't resolved for some reason (ie bug), we should still unfund it to get money back.